muikku-rest/src/main/java/fi/otavanopisto/muikku/rest/model/GuiderStudentRestModel.java

```diff
@@ -0,0 +1,239 @@
+package fi.otavanopisto.muikku.rest.model;
+
+import java.util.Date;
+import java.util.List;
+
+public class GuiderStudentRestModel extends Student {
```

Comment:
If you extend a class, you only need to introduce variable/getter/setter for the information you are adding to the base class. Now this class completely overwrites every variable that is already in the base class.